package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@teamsnap/teamsnap-ui",
-  "version": "4.0.0-alpha.110",
```

Comment:
Its fine to update this, but it shouldnt be necessary if the auto-deployment is working as expected.

Reply:
@jondauz weren't you and trey looking at the auto process not working? What came out of that